libs/FirmwareKit/tests/FirmwareKit_test.cpp

```diff
@@ -38,11 +39,14 @@ class FirmwareKitTest : public ::testing::Test
 	std::string bin_update_path				  = "/tmp/update-v2.0.0.bin";
 	std::array<uint8_t, 6> bin_update_content = {0x61, 0x62, 0x63, 0x64, 0x65, 0x66};	// "abcdef"
 
-	FirmwareVersion default_version = FirmwareVersion {1, 0, 0};
-	FirmwareVersion current_version = FirmwareVersion {1, 2, 3};
-	FirmwareVersion update_version	= FirmwareVersion {2, 0, 0};
+	// ? Default made sense when we were reading from a file in case the file was not readable
+	// ? With the #define OS_VERSION, this is not needed anymore
+	// TODO (@YannLocatelli): Remove the file related tests?
```

Comment:
About this one, as mentioned in https://github.com/leka/LekaOS/pull/1025#issue-1373247002, 

> Refactor getCurrentVersionFromFile

A big part of FirmwareKit should be dropped, the part using SD. In same time, most of tests will be dropped too.

Reply:
ah yes you're right! I've added back the `Needs` from #1025 to this PR description.

Reply:
See 

- #1072